firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -31,6 +31,45 @@ export interface FirestoreBigQueryEventHistoryTrackerConfig {
   tableId: string;
 }
 
+export const RawChangelogSchema: any = {
+  fields: [
+    {
+      name: "timestamp",
+      mode: "REQUIRED",
+      type: "TIMESTAMP",
+      description:
+        "The commit timestamp of this change in Cloud Firestore. If the operation is IMPORT, this timestamp is epoch to ensure that any operation on an imported document supersedes the IMPORT.",
+    },
+    {
+      name: "event_id",
+      mode: "REQUIRED",
+      type: "STRING",
+      description:
+        "The ID of the document change event that triggered the Cloud Function created by the extension. Empty for imports.",
```

Comment:
maybe:
imports -> IMPORT operations

Reply:
This sting is taken from our doc. Should we also change it there?

Reply:
I'm going to resolve this because this string is consistent with our doc.